### ISSUE TITLE
[BUG] Chart properties are populated, but returning null; works in example, doesn't work in own code

### ISSUE THREAD

**Issue by fantaandcode** [fenix-hub/godot-engine.easy-charts#78]
**Describe the bug**
I'm trying to create a line chart. When I use the example LineChart, it works. However, it doesn't work when I try to make my own chart and use the default ChartProperties.

**To Reproduce**
1. Create new scene
2. Add Chart node to root node
3. Create script for root node
4. Use following code:
```extends Control

@export var goods_info : Resource
@onready var chart : Chart = %Chart

var sell_f : Function
var buy_f : Function

var counter : int = 0

# Called when the node enters the scene tree for the first time.
func _ready():
	simulate_price() # sets price in the goods_info resource, sets buy and sell price
	
	sell_f = Function.new(
		PackedFloat32Array([-1, counter]),
		PackedFloat32Array([0, goods_info.sell_price]),
		'sell price',
		{ 
			color = Color("#36a2eb"),
			marker = Function.Marker.CIRCLE,
			type = Function.Type.LINE,
			interpolation = Function.Interpolation.STAIR
		}
	)
	
	buy_f = Function.new(
		PackedFloat32Array([-1, counter]),
		PackedFloat32Array([0, goods_info.buy_price]),
		'buy price',
		{ 
			color = Color("#36a2eb"),
			marker = Function.Marker.CIRCLE,
			type = Function.Type.LINE,
			interpolation = Function.Interpolation.STAIR
		}
	)
	
	chart.plot([sell_f, buy_f])

func _process(delta : float):
	chart.queue_redraw()

func simulate_price():
	goods_info.sim_price()
	var str_format = '$ %.2f'

func update_chart():
	counter += 1
	sell_f.add_point(counter, goods_info.sell_price)
	buy_f.add_point(counter, goods_info.buy_price)

func _minute_advance_update(amount):
	goods_info.update_counter += amount
	if goods_info.update_counter >= goods_info.update_frequency:
		goods_info.update_counter -= goods_info.update_frequency
		simulate_price()
	else:
		pass
```

**Expected behavior**
I expect a LineChart similar to the look of the example. The intent is to simulate the price at each tick and then have the chart updated.

**Screenshots**
![image](https://github.com/fenix-hub/godot-engine.easy-charts/assets/1066456/780778c1-0c3c-4d88-b5e7-ba1fe4fbc63e)

**Desktop (please complete the following information):**
 - OS: Linux
 - Godot Engine Version 4.1.1.stable.arch_linux
- Plugin Version v15.05.2023

**Additional context**
All examples work fine.

**Comment by fenix-hub** [comment]
First, thank you for using EC and sorry for answering this late.
Interestingly enough it seems that the `null` value is related to the `thene` variable and not the `font` one.I'm assuming this since the message says that you are not able to call the `set` function on a null instance.
Honestly I don't know why this is happening, I was not able to reproduce.
Could you please try this?
1. Create a new scene with a Control Node
2. Drag and drop (or instance using the "link button") the Chart.tscn scene
2. Create a simple Function object (x = [10, 20] y = [20, 30]) and plot it
![image](https://github.com/fenix-hub/godot-engine.easy-charts/assets/45271396/4a81fa32-487c-4368-8834-6ca9b0b729de)

**Comment by fenix-hub** [comment]
feel free to reach me out on discord and send a zip of your project if you need it.

**Comment by fantaandcode** [comment]
Hey, more than happy to use it. I was starting to create my own charts package, realized it was too much work, then decided to see if anybody was already creating one!

So I did as you said. I'm using the version: v18.08.23.

Here's how I set it up.
![image](https://github.com/fenix-hub/godot-engine.easy-charts/assets/1066456/385718eb-7671-4835-a454-61ef9d847c68)

![image](https://github.com/fenix-hub/godot-engine.easy-charts/assets/1066456/090f58b9-dfd3-4c8f-b979-787af20f04cb)

I'm using the code that you've specified above. I'm not sure what's wrong with them.
